### PR TITLE
Spanish translations

### PR DESCRIPTION
**Pull request by moonbeeper**

### CODE REVIEW

**Comment by moonbeeper** [comment]
yes es_ES translated

File: es-ES UNFINISHED.js [hunk 1]
```diff
@@ -1,210 +1,226 @@
...
 
 
     // OTHER
-    REQUESTED_BY: 'Requested by',
+    REQUESTED_BY: 'Solicitado Por',
     OK: 'OK',
     ERROR: 'ERROR'
-};
\ No newline at end of file
+    // yes this is bird
```
Comment (JustFlameBird) [inline es-ES UNFINISHED.js:224]: yes

**Comment by Roki100** [comment]
from what i can see you have ignored the auto fortmatting parts so like in `COOLDOWN` string it will say "segundoss" if there will be >1 seconds left
same about other such as `PREFIX_REMINDER` if there will be more than 1 prefix it will say "Prefix ActualLos Prefix en Este server Es: prefix 1, prefix2" which i dont think is proper

can you fix this please?
(and also remove "UNFINISHED" from the file name)

**Comment by moonbeeper** [comment]
yup sorry

**Comment by Roki100** [comment]
Thanks!

**Pull request merged**